auem/config/models/simplecnn.yaml

```diff
@@ -1,5 +1,5 @@
 model:
   class: auem.models.simplecnn.SimpleCNNBase
   params:
-    input_size: 282368
-    num_classes: 632
\ No newline at end of file
+    input_size: 42432
```

Comment:
It might be better long-term to have it calculate this, to make it less brittle?

Reply:
yup, we could calculate on the fly using these formulas? https://jovianlin.io/calculate-the-output-size-of-conv-layer/